### PR TITLE
GETT-88193 Add sandbox tests for payment method

### PR DESCRIPTION
**Pull request by maratori**

### CODE REVIEW

File: sandbox/payment_method_test.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+)
+
+func TestPaymentMethod(t *testing.T) {
+	t.Parallel()
+	client := GetClient(t)
+
+	t.Run("new & get & get-list", func(t *testing.T) {
+		t.Parallel()
+
+		customer := PrepareCustomer(t, client)
```
Comment (drscre) [inline sandbox/payment_method_test.go:20]: Should we cleanup the created resources after test is finished (ignoring any cleanup errors)? 
You can use t.Cleanup inside `PrepareXXX` methods.

**Review by drscre — CHANGES_REQUESTED** [review]

File: sandbox/payment_method_test.go (hunk 1 above)
Comment (maratori) [reply sandbox/payment_method_test.go:20]: Great question. Then we should clean up not only after `PrepareXXX` but after all tests.
Don't you mind if I do it in the next PR?

Comment (drscre) [reply sandbox/payment_method_test.go:20]: ok

**Review by drscre — APPROVED** [review]

File: sandbox/payment_method_test.go (hunk 1 above)
Comment (maratori) [reply sandbox/payment_method_test.go:20]: I think we should cleanup nothing.
We can't delete:
* tokens
* payments
* transactions

We can delete only:
* customers
* their payment methods

But why if we can't clean up all entities?

Comment (drscre) [reply sandbox/payment_method_test.go:20]: makes sense

**Pull request merged**